Objects/iterobject.c

```diff
@@ -427,20 +435,28 @@ anextawaitable_proxy(anextawaitableobject *obj, char *meth, PyObject *arg) {
 
 
 static PyObject *
-anextawaitable_send(anextawaitableobject *obj, PyObject *arg) {
+anextawaitable_send(PyObject *op, PyObject *arg)
+{
+    anextawaitableobject *obj = anextawaitableobject_CAST(op);
     return anextawaitable_proxy(obj, "send", arg);
 }
 
 
 static PyObject *
-anextawaitable_throw(anextawaitableobject *obj, PyObject *arg) {
-    return anextawaitable_proxy(obj, "throw", arg);
+anextawaitable_throw(PyObject *op, PyObject *args)
+{
+    anextawaitableobject *obj = anextawaitableobject_CAST(op);
+    return anextawaitable_proxy(obj, "throw", args);
 }
 
 
 static PyObject *
-anextawaitable_close(anextawaitableobject *obj, PyObject *arg) {
-    return anextawaitable_proxy(obj, "close", arg);
+anextawaitable_close(PyObject *op, PyObject *args)
+{
+    // TODO(picnixz): investigate why close() is marked with METH_VARARGS
+    // despite its docs indicating that it should be using METH_NOARGS.
```

Comment:
IMO it's just the docstring which is wrong, it should be `close(*args)`.

Reply:
I'll remove the TODO and will rebase https://github.com/python/cpython/pull/131671 on main later.